runtime/altair/src/lib.rs

```diff
@@ -1106,6 +1110,82 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
 
+// Block Rewards
+
+#[derive(Clone, Copy, PartialEq, Eq, Encode, Decode, TypeInfo, MaxEncodedLen, RuntimeDebug)]
+#[cfg_attr(
+	feature = "std",
+	derive(frame_support::Serialize, frame_support::Deserialize)
+)]
+pub enum RewardDomain {
+	Liquidity,
+	Block,
+}
+
+frame_support::parameter_types! {
+	// BlockRewards have exactly one group and currency
+	#[derive(scale_info::TypeInfo)]
+	pub const SingleCurrencyMovement: u32 = 1;
+	#[derive(scale_info::TypeInfo, Debug, PartialEq, Eq, Clone)]
+	pub const MaxChangesPerEpoch: u32 = 50;
+	pub const RewardsPalletId: PalletId = PalletId(*b"d/reward");
```

Comment:
Out of curiosity, why `d/` in `d/reward`?

Reply:
The ID please into the `types` crate and to ids.

Reply:
Shouldn't it rather be a primitives constant?

Reply:
We already have an ids mod in the types. I am not sure why I did not move it to the primitives. I guess my reasoning was that it is not a primitive to the logic of our chain. But no feelings where it lives, just want them grouped so that we notice when we change them.